### ISSUE TITLE
Auto updating mods feature.

### ISSUE THREAD

**Issue by mateowoetam** [PrismLauncher/PrismLauncher#272]
### Role

I am always tinkering with mods and some of them have update warnings

### Suggestion

Add a feature that allows auto updating mods, also wiould be useful if it had a toggle for all mods, as well as overrides in a per-mod case

### Benefit

I would no longer have to go into the files or get update messages every time I launch an instance, and it woluld save time

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

A feature that adds a toggle to allow auto-updates for mods would work per instance but can override/disable it in a per-mod case, it would also it could either run every time you open PrismLauncher or every time an instance is Launched, then it would look for updates on mods (using either Modrinth, Curseforge, or Ideally the Github source), this would make it's so that everything is always up to date and it would remove all those mod update warnings (e.g. Xaeros minimap has a very invasive sign where it covers the whole screen at launch) and would save time and effort because I always launch the game, see there is an update available and have to close the game, change the mod version, and re-launch (it would also be neat if it knows which mods had which dependencies and installed them)

**Comment by ghost** [comment]
use a mod like https://modrinth.com/mod/minecraft-mod-updater maybe

i hope xaerosm actually has a toggle for the update notification

**Comment by DioEgizio** [comment]
yeah there was an issue about this on polymc too (https://github.com/PolyMC/PolyMC/issues/933)

**Comment by mateowoetam** [comment]
It just uninstalled all my mods :(, something is very wrong with that mod.

**Comment by Ujhhgtg** [comment]
Sadly [minecraft-mod-updater](https://modrinth.com/mod/minecraft-mod-updater) was discontinued (and it does not work for me).

**Comment by ghost** [comment]
can't you just uh do 3 more clicks every few days
